doc/content/installation.md

```diff
@@ -0,0 +1,41 @@
+# Installation
+
+fillname requires a Python version between 3.8 and 3.10 (recomend 3.10)
+
+You can check a successfull instalaltion by running
+
+```shell
+fillname -h
+```
+
+## Installing with pip 
+
+
+The python fillname package can be found [here](https://pypi.org/project/fillname/).
```

Comment:
The `init.py` script should probably also replace fillname here and in other documentation files.